home-work-angular/app-cat/app.mock.service.js

```diff
@@ -0,0 +1,28 @@
+﻿(function () {
+    angular.module('catapp.mockData', ['ngMockE2E']);
+    angular.module('catapp').requires.push('catapp.mockData');
+
+    angular.module('catapp.mockData').run(run);
+    run.$inject = ['$httpBackend', 'voteCatService'];
+
+    function run($httpBackend, crudFactory) {
+        $httpBackend.whenGET('/partials/partial.login.html').passThrough();
```

Comment:
Instead of defining each view, better to use regex pattern to match the all the views